Data/models/PlaylistFolder.swift

```diff
@@ -83,6 +83,21 @@ final public class PlaylistFolder: NSManagedObject, CRUD, Identifiable {
         folder.delete()
     }
     
+    public static func updateFolder(folderID: NSManagedObjectID, _ update: @escaping (PlaylistFolder?, Error?) -> Void) {
+        DataController.perform(context: .new(inMemory: false), save: true) { context in
+            do {
+                guard let folder = try context.existingObject(with: folderID) as? PlaylistFolder else {
+                    update(nil, "Invalid Folder")
```

Comment:
Does this need to be localized?

Reply:
No need. This is only `logged`. The error alert shown is localized :)